8/debian10/8.9/Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN set -x \
     hkp://keyserver.ubuntu.com:80 \
     pgp.mit.edu \
   ; do \
-    gpg --no-tty --keyserver $server --recv-keys $GOSU_GPG \
+    gpg --no-tty --keyserver keys.openpgp.org --recv-keys $GOSU_GPG \
```

Comment:
Doesn't make sense to go in loop over other keyservers to retrieve from this hardcoded one every time.
Remove loop from template and leave bare command 
`gpg --no-tty --keyserver keys.openpgp.org --recv-keys $GOSU_GPG`

Reply:
On second thought, this change is not needed at all